web/webkit/src/test/scala/net/liftweb/http/HtmlNormalizerSpec.scala

```diff
@@ -153,7 +153,7 @@ class HtmlNormalizerSpec extends Specification with XmlMatchers with Mockito {
 
       js.toJsCmd must be matching("""(?s)\Qlift.onEvent("lift-event-js-\E[^"]+\Q","event",function(event) {doStuff;});
         |lift.onEvent("hello","event",function(event) {doStuff2;});
-        |lift.onEvent("lift-event-js-\E[^"]+\Q","event",function(event) {doStuff3;});\E""".stripMargin('|').lines.mkString("\n").r
+        |lift.onEvent("lift-event-js-\E[^"]+\Q","event",function(event) {doStuff3;});\E""".stripMargin('|').linesIterator.mkString("\n").r
```

Comment:
This turns out to be because of this guy: https://github.com/scala/bug/issues/11125

A note from that issue:

> note that it isn't safe to rely on the compiler to catch bad uses of .lines, you want to search your source code and find all of them
> 
> yes, often the compiler will catch it, e.g. s.lines.toVector fails to compile because Java's Stream doesn't have toVector
> 
> but more devious is something like s.lines.filter(...) == .... Java's Stream has filter, and == isn't type-safe, so if you're lucky you'll get a "are unrelated: they will most likely never compare equal" warning ... if you're lucky... and if you're lucky enough to spot the warning in your code's big pile of warnings it already generates that you tune out now...
> 
> I guess a Scalafix rewrite could help here, and wouldn't have false positives if you are using .lines method on something other than a String

Reply:
Oy vey.

Reply:
Oh I just realized this probably means we've got to hunt all uses of lines down. ffffff

Reply:
It feels like I can pick this particular issue up in the 2.13 PR, since I *think* that'll catch issues? Not sure.